_po4a-tools/po4a-add-language.sh

```diff
@@ -8,38 +8,40 @@ SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$SCRIPT_DIR"
 
 # Create and populate .po file folder
-mkdir ../_translator-files/po/$new_lang
+mkdir ../_translator-files/po/$lang_code
```

Comment:
Having a bad feeling about not quoting this:
```suggestion
mkdir ../_translator-files/po/"$lang_code"
```

may work?

Reply:
The style is different in the main repo though (and probably safer). We're way more strict there. See e.g. https://github.com/jamulussoftware/jamulus/blob/main/.github/autobuild/linux_deb.sh

Reply:
Surely `$lang_code` is semantically and syntactically incorrect if it contains spaces, so the quotes are redundant?

Reply:
Yes. The regex should not match in this case.

Reply:
```suggestion
mkdir "../_translator-files/po/${lang_code}"
```
would be more in keeping with the main repo, if we're going to refactor.

Reply:
The style in this PR just follows that used in the rest of the po4a scripts. I think a clean-up of all the scripts should happen in a separate PR and do them all at once.

Reply:
Ok. Let's leave this for another PR